homeassistant/components/unifiprotect/event.py

```diff
@@ -26,19 +32,39 @@ class ProtectEventEntityDescription(ProtectEventMixin, EventEntityDescription):
 
 EVENT_DESCRIPTIONS: tuple[ProtectEventEntityDescription, ...] = (
     ProtectEventEntityDescription(
-        key="doorbell",
-        translation_key="doorbell",
-        name="Doorbell",
+        key="ring",
```

Comment:
```suggestion
        key="doorbell",
```

Reply:
Lets keep the translation key the same as well so translators don't have to translate again